geolib/models/dsheetpiling/dsheetpiling_model.py

```diff
@@ -385,8 +386,7 @@ def set_construction(
 
         Args:
             top_level: Top level of the sheet piling.
-            elements: List of sheet piling elements, can be Sheet, DiaphragmWall, or Pile.
-            Elements are sorted on sheetpilingelementlevel.
+            elements: List of sheet piling elements, can be Sheet, DiaphragmWall, or Pile Elements are sorted on sheetpilingelementlevel.
```

Comment:
Suggested improvement: 
elements: List of sheet piling elements, can be Sheet, DiaphragmWall, or Pile elements.
Elements are sorted on sheetpilingelementlevel.

Reply:
Changed according to suggested improvement